tests/h/api/storage_test.py

```diff
@@ -646,25 +386,6 @@ def test_it_deletes_the_annotation(self, fetch_annotation, mock_request):
 
         mock_request.db.delete.assert_called_once_with(first_return_value)
 
-    def test_it_fetches_the_legacy_annotation(self, fetch_annotation, mock_request):
-        storage.delete_annotation(mock_request, "test_id")
-
-        assert fetch_annotation.call_args == mock.call(mock_request,
-                                                       "test_id",
-                                                       _postgres=False)
-
-    def test_it_deletes_the_legacy_annotation(self, fetch_annotation, mock_request):
-        first_return_value = mock.Mock()
-        second_return_value = mock.Mock()
-        fetch_annotation.side_effect = [
-            first_return_value,
-            second_return_value,
-        ]
-
-        storage.delete_annotation(mock_request, "test_id")
-
-        second_return_value.delete.assert_called_once_with()
-
     @pytest.fixture
     def mock_request(self, session):
         request = DummyRequest()
```

Comment:
It's probably not necessary for this to be a fixture anymore, though maybe kind of tedious to change